src/hls-adapter.js

```diff
@@ -528,7 +525,7 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
    */
   _onManifestLoaded(): void {
     HlsAdapter._logger.debug('The source has been loaded successfully');
-    this._hls.startLoad();
+    this._hls.startLoad(this._startTime ? this._startTime : 0);
```

Comment:
Does `startPosition` not working at all? or setting it and calling `startLoad is the issue`?

I think in the way we do it now we simply cancel the option of hls.js to not start loading frags after manifest was loaded, cause we always call `startLoad` in `_onManifestLoaded`.
Also, as hls.js default is to autoStartLoad, I think doing it is not needed here?